core/tasks.rs

```diff
@@ -0,0 +1,273 @@
+use futures::task::AtomicWaker;
```

Comment:
```suggestion
// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.

use futures::task::AtomicWaker;
```

Reply:
I really need to set up copyright checker